monero-outreach-docs/translations/fr/archive/triptych_fr.md

```diff
@@ -0,0 +1,43 @@
+
+Les flux de Monero des clés publiques sont gardés confidentiels par l’usage d’un procédé de chiffrage nommé "cryptographic ring signatures". Ce procédé permet de dissimuler la clé publique d’un TXO en l'incorporant à un groupe de transaction-leurres (également nommée "mixins"). L’observation d’une transaction ne permettra donc pas de déterminer si un TXO donné est l’original ou non. Vous pouvez voir un exemple d’émission ou de réception de TXO en cherchant une transaction a l’aide d’un programme d’analyse de la chaine de blocs de Monero tel que [moneroblocks.info](https://moneroblocks.info/) ou [xmrchain.net](https://xmrchain.net/).
+
+La manière dont les transaction-leurres TXO sont sélectionnées a partir de la chaine de blocs s’est améliorée au fil du temps. Au tout début, l'intégration de transaction-leurres - bien que faisant partie du système - restait facultative, et les leurres étaient sélectionnées uniformément sur l'historique entière des transactions. Avec le temps, le nombre de transaction-leurres grandit et devint obligatoire, pour atteindre aujourd'hui exactement 10 transaction-leurres par ordre d’exécution. De même, le procédé statistique de sélection fut affiné au fil du temps pour lui donner plus de complexité grâce au "tailored random process" (un algorithme basé sur le procédé de sélection pour l’exécution de l’ordre d’après une théorie statistique de distribution gamma particulière, que nous utilisons encore a ce jour).
+
+Les algorithmes du "core ring signature" ont également été améliorés: la publication de proposition originelle ("whitepaper") de CryptoNote présentait un processus de signature de cercle différent de celui que nous employons actuellement, qui est bâti sur l’utilisation des signatures "Multilayered Linkable Spontaneous Anonymous Group" (MLSAG). Une amélioration substantielle, nommée signatures "Compact Linkable Spontaneous Anonymous Group" (CLSAG), est attendue prochainement, et promet de réduire d’environ un quart le poids (en termes de données) des transactions et d’un dixième leur temps de confirmation (pour une transaction de deux entrées et deux sorties).
+
+Toutes ces améliorations ont donné à Monero de puissantes capacités d’anonymisation. A chaque mouvement de Monero, tout TXO sortant prend la forme d’une clé publique TXO de couverture, sans lien cryptographique avec l’adresse émettrice, mêlée a 10 autres clé publiques-leurres, sélectionnées pour leur vraisemblance. Cela rend toute surveillance extrêmement difficile, tout en assurant une rapidité et une fluidité d’exécution du logiciel de Monero, et offrant une chaine de blocs de taille et de vitesse acceptable.
```

Comment:
Original sentence:
A chaque mouvement de Monero, tout TXO sortant prend la forme d’une clé publique TXO de couverture, sans lien cryptographique avec l’adresse émettrice, mêlée a 10 autres clé publiques-leurres, sélectionnées pour leur vraisemblance.
Suggested sentence:
A chaque mouvement de Monero, tout TXO sortant prend la forme d’une clé publique TXO (ou adresse furtive), sans lien cryptographique avec l’adresse émettrice. Cette clé publique est mêlée a 10 autres clés publiques-leurres, sélectionnées pour leur vraisemblance.  

Reason: The original english sentence is too long and it is better to split it in two. Furthermore, the term "clé publique TXO" is synonyme of "adresse furtive", such that adding 'couverture' is not necessary.

Reply:
Agreed. Integrated.